src/javascript/binary/base/page.js

```diff
@@ -523,6 +523,27 @@ Header.prototype = {
             BinarySocket.send({"logout": "1"});
         });
     },
+    
+    validate_cookies: function(){
+        function is_valid_id(id){
+            return /^(MX|MF|VRTC|MLT|CR|VRTJ|JP)[0-9]+$/i.test(id);
+        }
+        
+        if (getCookieItem('login') && getCookieItem('loginid_list')){
+            var accIds = $.cookie("loginid_list").split("+");
+            var loginid = $.cookie("loginid");
+                                    
+            if(!is_valid_id(loginid)){
+                BinarySocket.send({"logout": "1"});
+            }
+            
```

Comment:
for example, I can do client_form.is_loginid_valid(page.client.loginid) and it will return true or false